C/Cpp_include/c4Observer.hh

```diff
@@ -42,6 +44,7 @@ struct C4CollectionObserver : public fleece::InstanceCounted, C4Base {
 
     /// Metadata of a change recorded by C4CollectionObserver. (Equivalent to C4CollectionChange.)
     struct Change {
+        CollectionSpec collection;      ///< The collection the document belongs to
```

Comment:
It's a debatable point, but in the current API we are, just in case they want to assign a common implementation that varies slightly based on the collection.

Reply:
Why a CollectionSpec instead of a C4Collection?

Reply:
I think the reasoning was that it is easier to get a collection object on the platform side given its information rather than a pointer which would need to be linked later to a platform collection object.